### PR TITLE
[CES-18232] Make banner accessible

### PR DESCRIPTION
**Pull request by aurelie-forzani**
## Description

Make banner accessible
- action button: remove `outline: none;` 
- close button : 
  - use `button` balise
  - remove `top: toRem(2);` to center the button in the outline when the button is focused

## JIRA ticket

[CES-18232](https://perzoinc.atlassian.net/browse/CES-18232)

[CES-18232]: https://perzoinc.atlassian.net/browse/CES-18232?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by aurelie-forzani** [comment]
ok to test

File: packages/components/src/components/banner/Banner.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const Banner: React.FC<BannerProps> = ({
 
       {/* Close button (optional) */}
       {isClosable && onClose && (
-        <div className="tk-banner__close" onClick={onClose} />
+        <button className="tk-banner__close" onClick={onClose} />
```
Comment (benoit-sy) [inline packages/components/src/components/banner/Banner.tsx:47]: You would need to specify a `type="button"` here, as the default on eis `submit`.

Someone using the component inside a form would have it submitted when closing it.

**Review by benoit-sy — CHANGES_REQUESTED** [review]

File: packages/components/src/components/banner/Banner.tsx (hunk 1 above)
Comment (aurelie-forzani) [reply packages/components/src/components/banner/Banner.tsx]: updated, thanks !

**Comment by aurelie-forzani** [comment]
ok to test

**Pull request merged**